CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Unreleased
+
+### Breaking change
+
+- Renamed `enableTimeToFullDisplay` to `enableTimeToFullDisplayTracing` (#3106)
```

Comment:
Maybe we should point out that this is experimental API which can change at any time without a major revision. People not using it might still see "breaking change" and ignore the update.